docs(compute): instances/elastic-metal/dedibox datasheet (#2542)
scaleway/docs-content: compute/instances/reference-content/instances-datasheet.mdx

---
meta:
  title: Instances - Technical product specification
  description: This page shows the technical characteristics of the Instances offer.
content:
  h1: Instances - Technical product specification
  paragraph: This page shows the technical characteristics of the Instances offer.
tags: dedibox server
dates:
  validation: 2023-12-18
  posted: 2023-12-18
categories:
  - compute
---

> 

Author: ```suggestion

# Instances datasheet
The datasheet for Instances provides a concise overview of their performance, technical features, components, materials, and associated documentation.
```

---
meta:
  title: Instances - Technical product specification

>   description: This page shows the technical characteristics of the Instances offer.

Reviewer: ```suggestion
  description: This page shows the technical characteristics of the different Instance offers.
```

content:
  h1: Instances - Technical product specification

>   paragraph: This page shows the technical characteristics of the Instances offer.

Reviewer: ```suggestion
  paragraph: This page shows the technical characteristics of the different Instance offers.
```

tags: dedibox server
dates:
  validation: 2023-12-18
  posted: 2023-12-18
categories:
  - compute
---

# Instances datasheet

> The datasheet for Instances provides a concise overview of their performance, technical features, components, materials, and associated documentation.

Reviewer: ```suggestion
This datasheet provides a concise overview of the performance, technical features, components, materials, and associated documentation for different Instance offers.
```

---
meta:
  title: Instances datasheet
  description: This page shows the technical characteristics of the different Instance offers.
content:
  h1: Instances datasheet
  paragraph: This page shows the technical characteristics of the different Instance offers.
tags: dedibox server
dates:
  validation: 2023-12-18
  posted: 2023-12-18
categories:
  - compute
---

# Instances datasheet

This datasheet provides a concise overview of the performance, technical features, components, materials, and associated documentation for different Instance offers.

<Message type="note">
  Note that the **Max. NIC bandwidth** specifies the maximum available bandwidth for this Instance type. Bandwidth is shared between internet and Private Network traffic.

> 

Reviewer 2: ```suggestion
```

</Message>

<Concept>

## STARDUST1 Instances

 See below the technical specifications of Stardust Instances:

| Range            	  | Learning                                	|
|:------------------------|:------------------------------------------|

> | Instance Type     	| STARDUST1-S                             	|

Reviewer 3: ```suggestion
| Instance type     	| STARDUST1-S                             	|
```

| Availability Zone 	| PAR1, AMS1 and WAW2                     	|
| Storage           	| Local (10GB) or Block (from 10GB to 10TB) |
| Max. NIC bandwidth  | 100 Mbit/s              	                |

> | CPU Type          	| AMD EPYC 7282 (2,8 GHz) or 7281 (2,1 GHz)	|

Reviewer 3: ```suggestion
| CPU type          	| AMD EPYC 7282 (2,8 GHz) or 7281 (2,1 GHz)	|
```

| CPU architecture  	| `amd64`                                  	|
| Resources         	| Shared vCPUs                            	|
| Sizing          	  | 1 vCPU, 1 GiB RAM                         |
| vCPU:RAM ratio    	| 1:1                     	                |
| SLA               	| None                    	                |

</Concept>

<Concept>

## Developement and General Purpose Instances

See below the technical specifications of Development and General Purpose Instances:

| Range                         	| Cost-Optimized                                                              	|
|:--------------------------------|:------------------------------------------------------------------------------|

> | Instance Type   	              | DEV1 <br /> GP1                                                      	        |

Reviewer 3: ```suggestion
| Instance type   	              | DEV1 <br /> GP1                                                      	        |
```

| Availability Zone   	          | PAR1, PAR2, PAR3 (excl. DEV1), AMS1, AMS2, WAW1, WAW2                         |
| Storage   	                    | Local (up to 150GB) or Block (from 10GB to 10TB)                                            |
| Max. NIC bandwidth   	          | From 200 to 500 Mbps                                                          |

> | CPU Type   	                    | DEV1: AMD EPYC 7281 (2,1 GHz) or equivalent<br />GP1: AMD EPYC 7410P (2 GHz) or equivalent|

Reviewer 3: ```suggestion
| CPU type   	                    | DEV1: AMD EPYC 7281 (2,1 GHz) or equivalent<br />GP1: AMD EPYC 7410P (2 GHz) or equivalent|
```

| CPU architecture  	            | `amd64`                                                                     	|
| Resources   	                  | Shared vCPUs                                                                  |
| Sizing   	                      | From 2 to 4 vCPUs <br />From 2 to 12 GiB RAM                                   |
| vCPU:RAM ratio                  | Various<br />(1:1, 1:2, 1:3)                                                  |

</Concept>

<Concept>

## PLAY2 and PRO2 Instances

See below the technical specifications of PLAY2 and PRO2 Instances:

| Range                         	| Cost-Optimized                                                              	|
|:--------------------------------|:------------------------------------------------------------------------------|

> | Instance Type   	              | PLAY2 <br /> PRO2                                                    	        |

Reviewer 3: ```suggestion
| Instance type   	              | PLAY2 <br /> PRO2                                                    	        |
```

| Availability Zone   	          | PAR1, PAR2, PAR3 (excl PLAY2), AMS1, AMS2, AMS3, WAW1, WAW2, WAW3             |
| Storage   	                    | Block (from 10 GB to 10 TB)                                                  |
| Max. NIC bandwidth   	          | From 100 Mbps  to 6 Gbps                                                      |

> | CPU Type   	                    | AMD EPYC 7543 (2,8 GHz)                                                       |

Reviewer 3: ```suggestion
| CPU type   	                    | AMD EPYC 7543 (2,8 GHz)                                                       |
```

| CPU architecture  	            | `amd64`                                                                     	|
| Resources   	                  | Shared vCPUs                                                                  |
| Sizing   	                      | From 1 to 32 vCPUs <br />From 2 to 128 GiB RAM                                |
| vCPU:RAM ratio                  | Various<br />(1:2, 1:4)                                                       |

</Concept>

<Concept>

## COP-ARM Instances

The table below displays the technical specifications of COP-ARM Instances:

| Range                         	| Cost-Optimized                                                              	|
|:--------------------------------|:------------------------------------------------------------------------------|

> | Instance Type   	              | COPARM1                                                             	        |

Reviewer 3: ```suggestion
| Instance type   	              | COPARM1                                                             	        |
```

| Availability Zone   	          | PAR2                                                                          |
| Storage   	                    | Block (from 10 GB to 10 TB)                                                   |
| Max. NIC bandwidth   	          | From 200 Mbps to 3.2 Gbps                                                     |

> | CPU Type   	                    | ARM (Ampere Altra Max M128-30)                                                |

Reviewer 3: ```suggestion
| CPU type   	                    | ARM (Ampere Altra Max M128-30)                                                |
```

| CPU architecture  	            | `arm64`                                                                     	|
| Resources   	                  | Shared vCPUs                                                                  |
| Sizing   	                      | From 2 to 128 vCPUs <br />From 8 to 128 GiB RAM                               |
| vCPU:RAM ratio                  | 1:4                                                                           |

</Concept>

<Concept>

## Enterprise Instances

See below the technical specifications of Enterprise Instances:

| Range                         	| Production-Optimized                                                          |
|:--------------------------------|:------------------------------------------------------------------------------|

> | Instance Type   	              | ENT1                                                                	        |

Reviewer 3: ```suggestion
| Instance type   	              | ENT1                                                                	        |
```

| Availability Zone   	          | PAR1, PAR2, PAR3, AMS1, AMS2, AMS3, WAW1, WAW2, WAW3                          |
| Storage   	                    | Block (from 10 GB to 10 TB)                                                   |
| Max. NIC bandwidth   	          | From 400 Mbps to 20 Gbps                                                      |

> | CPU Type   	                    | AMD EPYC 7543 (2,8 GHz)                                                       |

Reviewer 3: ```suggestion
| CPU type   	                    | AMD EPYC 7543 (2,8 GHz)                                                       |
```

| CPU architecture  	            | `amd64`                                                                     	|
| Resources   	                  | Dedicated vCPUs*                                                              |
| Security feature                | Secure Encrypted Virtualization                                               |
| Sizing   	                      | From 2 to 96 vCPUs  <br /> From 8 GiB to 384 GiB RAM                          |
| vCPU:RAM ratio                  | 1:4                                                                           |

> \* Instances with dedicated vCPU do not share their compute resources with other Instances (1 vCPU = 1 CPU thread dedicated to that Instance). This type of Instance is particularly recommended for running production-grade compute-intensive applications

Reviewer 3: Do we add a full stop at the end of this sentence? (same for the one below)

Author: ```suggestion
\* Instances with dedicated vCPU do not share their compute resources with other Instances (1 vCPU = 1 CPU thread dedicated to that Instance). This type of Instance is particularly recommended for running production-grade compute-intensive applications.
```

</Concept>

<Concept>

## Production-Optimized Instances

See below the technical specifications of Production-Optimized Instances:

| Range                         	| Production-Optimized                                                          |
|:--------------------------------|:------------------------------------------------------------------------------|

> | Instance Type   	              | POP2                                                                	        |

Reviewer 3: ```suggestion
| Instance type   	              | POP2                                                                	        |
```

| Availability Zone   	          | PAR1, PAR2, PAR3, AMS1, AMS2, AMS3, WAW2, WAW3                                |
| Storage   	                    | Block (from 10 GB to 10 TB)                                                   |
| Max. NIC bandwidth   	          | From 400 Mbps to 20 Gbps                                                      |

> | CPU Type   	                    | AMD EPYC 7543 (2,8 GHz)                                                       |

Reviewer 3: ```suggestion
| CPU type   	                    | AMD EPYC 7543 (2,8 GHz)                                                       |
```

| CPU architecture  	            | `amd64`                                                                     	|
| Resources   	                  | Dedicated vCPUs*                                                              |
| Sizing   	                      | From 2 to 96 vCPUs  <br /> From 8 GiB to 384 GiB RAM                          |
| vCPU:RAM ratio                  | 1:4                                                                           |

> \* Instances with dedicated vCPU do not share their compute resources with other Instances (1 vCPU = 1 CPU thread dedicated to that Instance). This type of Instance is particularly recommended for running production-grade compute-intensive applications

Author: ```suggestion
\* Instances with dedicated vCPU do not share their compute resources with other Instances (1 vCPU = 1 CPU thread dedicated to that Instance). This type of Instance is particularly recommended for running production-grade compute-intensive applications.
```

</Concept>

<Concept>

## Workload-Optimized Instances

See below the technical specifications of Workload-Optimized Instances:

* High-Memory: Designed for RAM-intensive usages, and high memory production applications, these machines provide more RAM than vCPU.
* High-CPU: Made for high computing workloads and compute-bound applications, this machine provides more vCPU than RAM.

| Range                         	| Workload-Optimized                                                            |
|:--------------------------------|:------------------------------------------------------------------------------|

> | Instance Type   	              | POP2-HM, POP2-HC                                                      	      |

Reviewer 3: ```suggestion
| Instance type   	              | POP2-HM, POP2-HC                                                      	      |
```

| Availability Zone   	          | PAR1, PAR2, PAR3, AMS1, AMS2, WAW2, WAW3                                      |
| Storage   	                    | Block (from 10 GB to 10 TB)                                                   |
| Max. NIC bandwidth   	          | From 400 Mbps to 12.8 Gbps                                                    |

> | CPU Type   	                    | AMD EPYC 7543 (2,8 GHz)                                                       |

Reviewer 3: ```suggestion
| CPU type   	                    | AMD EPYC 7543 (2,8 GHz)                                                       |
```
